c2corg_api/tests/emails/test_emails.py

```diff
@@ -0,0 +1,53 @@
+import os
+import unittest
+
+from pyramid.paster import get_appsettings
+from pyramid_mailer import get_mailer
+
+from c2corg_api import main
+from c2corg_api.emails.email_service import EmailService
+from webtest import TestApp
+
+from nose.plugins.attrib import attr
+
+from c2corg_api.models.user import User
+
+curdir = os.path.dirname(os.path.abspath(__file__))
+configfile = os.path.realpath(os.path.join(curdir, '../../../test.ini'))
+settings = get_appsettings(configfile)
+
+
+class EmailTests(unittest.TestCase):
+    def setUp(self):
+        app = main({}, **settings)
+        self.app = TestApp(app)
```

Comment:
You don't want to use `BaseTestCase` (which already configures `TestApp`) because you want to avoid to set up the database?